src/tree/SlotTreeItemBase.ts

```diff
@@ -166,4 +168,23 @@ export abstract class SlotTreeItemBase extends AzureParentTreeItem<ISiteTreeRoot
     public async deleteTreeItemImpl(): Promise<void> {
         await deleteSite(this.root.client);
     }
+
+    public showCreatedOutput(): void {
+        const slot: string = localize('slot', 'slot');
+        const functionApp: string = localize('functionApp', 'function app');
+        const createdNewSlotTree: string = localize('createSlotTree', 'Created new {0} "{1}": {2}', this.root.client.isSlot ? slot : functionApp, this.root.client.fullName, `https://${this.root.client.defaultHostName}`);
```

Comment:
So it's fine to insert things like a website name into a localized string since that won't be translated, but it's generally discouraged to insert things like "slot" which could be translated. For example, some languages might put the adjective after the noun like "Created slot new", which wouldn't work in your current setup. Just duplicate the string